themes/default/content/docs/pulumi-cloud/esc/providers/vault-login.md

```diff
@@ -24,6 +24,62 @@ The `vault-login` provider enables you to log in to HashiCorp Vault using OpenID
           role: example-role
 ```
 
+## Configuring OIDC in Vault
+To add Pulumi Cloud as an OIDC provider in Vault you'll need to do the following:
+- Enable the [`JWT` auth method](https://developer.hashicorp.com/vault/docs/auth/jwt)
+- Create an associated role
+
+### Enable JWT Auth Method
+To enable the [JWT auth method](https://developer.hashicorp.com/vault/docs/auth/jwt) in Vault:
+```
+$ vault auth enable -path=jwt jwt
+Success! Enabled jwt auth method at: jwt/
+
+$ vault write auth/jwt/config \
+    oidc_discovery_url="https://api.pulumi.com/oidc" \
+    bound_issuer="https://api.pulumi.com/oidc"
+Success! Data written to: auth/jwt/config
+```
+
+### Create JWT Role
+To create a [role](https://developer.hashicorp.com/vault/api-docs/auth/jwt#create-update-role) (assumes existing policy `myExamplePolicy`) using the CLI:
```

Comment:
```suggestion
To create a [role](https://developer.hashicorp.com/vault/api-docs/auth/jwt#create-update-role) (assumes existing policy `myExamplePolicy`) using the CLI:
```

Would you mind including the steps to create a policy? It can be a simple one with a note for the user to make sure the policy includes the permissions that they need for their own usage.

Reply:
Good idea, I've moved this @toriancrane